### PR TITLE
[projects] add check for createProject

### PR DESCRIPTION
**Pull request by AlexTugarev**

### CODE REVIEW

**Comment by geropl** [comment]
@AlexTugarev is this still relevant/already done?

**Comment by AlexTugarev** [comment]
@geropl, yes this is very relevant. I just updated the branch.

File: components/dashboard/src/projects/NewProject.tsx
Comment (geropl) [inline components/dashboard/src/projects/NewProject.tsx:181]: Had to blink twise :slightly_smiling_face:

**Review by geropl — APPROVED** [review]
Code LGTM

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: d85c125210e0c7451447a8e6dc54e40c668799cd</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/5421#pullrequestreview-753706734" title="Approved">geropl</a>*

*No associated issue*. Update pull-request body to add a reference to an issue, or get approval with `/approve no-issue`

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[components/dashboard/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/dashboard/OWNERS)~~ [geropl]
- ~~[components/server/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/server/OWNERS)~~ [geropl]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/5421#issuecomment-919025541" title="Approved">AlexTugarev</a>*, *<a href="https://github.com/gitpod-io/gitpod/pull/5421#pullrequestreview-753706734" title="Approved">geropl</a>*

Associated issue requirement bypassed by: *<a href="https://github.com/gitpod-io/gitpod/pull/5421#issuecomment-919025541" title="Approved">AlexTugarev</a>*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[components/dashboard/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/dashboard/OWNERS)~~ [AlexTugarev,geropl]
- ~~[components/server/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/server/OWNERS)~~ [AlexTugarev,geropl]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
